### DISCUSSION TITLE
How to "clean up" old certs?

### DISCUSSION THREAD

**Discussion by mrfuntime** [NginxProxyManager/nginx-proxy-manager#2425] [Q&A]
Think I found the answer to my question; sorry for the intrusion.

**Comment by tedlefter** [comment]
Consider posting the answer you found, others could find it useful.

**Reply by tedlefter** [reply]
I believe this is a side effect of how certbot operates.
https://community.letsencrypt.org/t/how-do-i-clear-the-old-data-from-the-etc-letsencrypt-directory/146667

Looks like this was recently improved upon.
https://github.com/certbot/certbot/issues/4635
